src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -52,6 +52,11 @@ public static bool ShouldHideAdvancedMembers(this Document document)
             foreach (var linkedDocumentId in linkedDocumentIds)
             {
                 var linkedDocument = document.Project.Solution.GetDocument(linkedDocumentId);
+                if (!linkedDocument.SupportsSyntaxTree)
```

Comment:
I'm not sure about this one.  This seems like a generic item to just process over documents/linked documents.  I don't see anything about it that would be specific to documents with syntax trees.